src/components/forms/common/ChoiceDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import type { FC, ReactNode } from 'react';
+import {
+	Button,
+	Dialog,
+	DialogActions,
+	DialogContent,
+	DialogContentText,
+	DialogTitle
+} from '@mui/material';
+
+type Props = {
+	name: string;
+	content: ReactNode;
+	open: boolean;
+	handleClose: () => void;
+	handleActionOnYes: () => void;
+};
+
+export const ChoiceDialog: FC<Props> = ({
+	name,
+	content,
+	open,
+	handleClose,
+	handleActionOnYes
+}) => (
+	<Dialog open={open} onClose={handleClose}>
+		<DialogTitle sx={{ fontWeight: 'bold' }}>{name}</DialogTitle>
+		<DialogContent>
+			<DialogContentText sx={{ color: 'text.primary' }}>
+				{content}
+			</DialogContentText>
+		</DialogContent>
+		<DialogActions>
+			<Button
+				variant="contained"
+				sx={{ backgroundColor: 'primary.light' }}
+				onClick={handleActionOnYes}
+			>
+				Yes
```

Comment:
Z UX hlediska bychom měli přetextovat tlačítka na něco jako "Delete" a "Cancel". Současná tlačítka jsou odpovědí na otázku výše, ale po přečtení těch tří vět uživatel může zapomenout, jak přesně zněla otáka, a tak bude "Delete" a "Cancel" vhodnější.

A ještě detail (omlouvám se, že tak šťourám). Možná bych tlačítka prohodil, ať to máme stejně jako i jinde v aplikaci (kde máme např. vlevo "Back" a vpravo jednotlivé akce). Tlačítku "Delete" bych pak dal tu primární barvu a "Cancel" tu `primary.light`.

Reply:
S barvami souhlasím, nejsem si ale jistý přetextováním: potom už ten dialog totiž nepůjde použít na nic jiného, leda bychom dali do propsů ten text pro oba buttony, což vlastně můžeme.

Reply:
Stačil by jenom label tlačítka pro akci. "Cancel" by měl platit pro vše.